single_kernel_mongo/managers/mongodb_operator.py

```diff
@@ -508,18 +519,10 @@ def on_update_status(self) -> None:
 
         try:
             self.perform_self_healing()
-        except ServerSelectionTimeoutError:
-            deployment = (
-                "replica set" if self.state.is_role(MongoDBRoles.REPLICATION) else "cluster"
-            )
-            self.charm.status_manager.to_waiting(
-                f"Waiting to sync internal membership across the {deployment}"
-            )
-        else:
-            self.charm.status_manager.to_active(None)
```

Comment:
@Gu1nness is this done elsewhere?

Reply:
This is now in mongo.py (as a part of `get_status`)